src/components/DayPicker/DayPicker.tsx

```diff
@@ -0,0 +1,345 @@
+import * as React from 'react';
+import clsx from 'clsx';
+import ReactDayPicker, { LocaleUtils } from 'react-day-picker';
+// Is this safe? consumers might not be using webpack w/ CSS loaders,
+import 'react-day-picker/lib/style.css';
+import { Calendar, ChevronLeft, ChevronRight } from '@lifeomic/chromicons';
+
+import { makeStyles, useTheme } from '../../styles';
+import { TextField, TextFieldProps } from '../TextField';
+import { GetClasses } from '../../typeUtils';
+import { ButtonUnstyled } from '../ButtonUnstyled';
+
+export const DayPickerStylesKey = 'ChromaDayPicker';
+export type DayPickerClasses = GetClasses<typeof useStyles>;
+
+const setToMidnight = (date: Date) => {
+  date.setHours(0);
+  date.setMinutes(0);
+  date.setSeconds(0);
+  date.setMilliseconds(0);
+};
+
+/**
+ * Helper hook for handling clicks outside of provided `ref` node.
+ */
+const useClickOutside = (
+  ref: React.RefObject<Node>,
+  handler: () => void,
+  dependencies: any[]
+) => {
+  React.useEffect(() => {
+    const handleClickOutside = (e: any) => {
+      if (ref.current?.contains(e.target)) {
+        return;
+      }
+      handler();
+    };
+
+    window.addEventListener('mousedown', handleClickOutside);
+
+    return () => {
+      window.removeEventListener('mousedown', handleClickOutside);
+    };
+  }, [ref.current, ...dependencies]);
+};
+
+const useStyles = makeStyles((theme) => ({
+  dayPicker: {
+    position: 'absolute',
+    zIndex: 99999,
+    marginTop: theme.spacing(1),
+    padding: theme.spacing(2),
+    borderRadius: theme.spacing(1.5),
```

Comment:
Can we use `borderRadius: theme.spacing (1.25)` here to be consistent with other popovers?

Reply:
Done!